test/Plugin.test.js

```diff
@@ -0,0 +1,56 @@
+const chai = require( 'chai' );
+const expect = chai.expect;
+const Plugin = require( '../src/Plugin' );
+const EmptyPlugin = require( './fixtures/EmptyPlugin' );
+const CompletePlugin = require( './fixtures/CompletePlugin' );
+describe( 'Plugin', () => {
+
+	it( 'is a class', () => {
+		expect( Plugin ).to.be.a( 'function' );
+	} );
+
+	describe( 'configure', () => {
+
+		it( 'is a function', () => {
+			expect( new CompletePlugin().configure ).to.be.a( 'function' );
+		} );
+
+		it( 'throw an Error when not inherited', () => {
+			expect( new EmptyPlugin().configure ).to.throw();
+		} );
+
+		it( 'not to throw an Error() when inherited', () => {
+			expect( new CompletePlugin().configure ).to.not.throw();
+		} );
+
+	} );
+
+	describe( 'execute', () => {
+
+		it( 'is a function', () => {
+			expect( new CompletePlugin().execute ).to.be.a( 'function' );
```

Comment:
This test does not test the appropriate `execute` method. It should check if `Plugin.prototype.execute` is a function.